Refactor (#6)
libdemdigital/security-advice: why.md

---
title: Our reasoning
---

# Credits:

> This is a lightly edited version of Tech Solidarity's security guideline for Congressional campaigns, with input from Motherboard's Guide to Not Getting Hacked, and @swiftonsecurity's decentsecurity.com. Changes include making it easier to follow when there has been no in-person training accompanying it.

Reviewer: ```suggestion
This is based heavily on Tech Solidarity's security guideline for Congressional campaigns, with input from Motherboard's Guide to Not Getting Hacked, and @swiftonsecurity's decentsecurity.com. Changes include making it easier to follow when there has been no in-person training accompanying it.
```

(the editing is no longer so light 😆 )
